src/build/mod.rs

```diff
@@ -13,6 +13,9 @@ pub fn build(target: &Target) -> Result<(), failure::Error> {
             message::info("JavaScript project found. Skipping unnecessary build!")
         }
         TargetType::Rust => {
+            let _ = which::which("rustc")
+                .map_err(|e| failure::format_err!("'rustc' not found: {}", e))?;
```

Comment:
```rust
use wrangler::terminal::styles;
```

```rust
let rust_url = styles::url("https://www.rust-lang.org/tools/install");
```

```suggestion
                .map_err(|e| failure::format_err!("'rustc' not found, installation documentation can be found here: {}, rust_url))?;
```

Reply:
lets point the devs in the right direction for installation here just so it's easy for them to find.

Reply:
It might be good to still print the error message. Maybe:
```
                .map_err(|e| failure::format_err!(
"'rustc' not found: {}. Installation documentation can be found here: {}",
e, styles::url("https://www.rust-lang.org/tools/install")))?;
```

Reply:
what does the error message look like when it's printed out there? can you give an example?

Reply:
I believe it'd look something like:
`'rustc' not found: cannot find binary path. Installation documentation can be found here: https://www.rust-lang.org/tools/install`

Where "cannot find binary path" would be any of the messages from `which::Error`
```rust
#[derive(thiserror::Error, Copy, Clone, Eq, PartialEq, Debug)]
pub enum Error {
    #[error("bad absolute path")]
    BadAbsolutePath,
    #[error("bad relative path")]
    BadRelativePath,
    #[error("cannot find binary path")]
    CannotFindBinaryPath,
    #[error("cannot get current directory")]
    CannotGetCurrentDir,
    #[error("cannot canonicalize path")]
    CannotCanonicalize,
}
```

Reply:
👍 sounds good to me